libraries/elixir/lib/standard_webhooks.ex

```diff
@@ -124,7 +124,7 @@ defmodule StandardWebhooks do
   end
 
   defp sign_and_encode(to_sign, secret) do
-    :crypto.mac(:hmac, :sha256, secret, to_sign)
+    :crypto.mac(:hmac, :sha256, Base.decode64!(secret), to_sign)
```

Comment:
Thanks a lot for the PR!

There's another small bug in this code, can you potentially fix that too? It should also strip the `whsec_` prefix if one exists. E.g. see the JavaScript code: https://github.com/standard-webhooks/standard-webhooks/blob/main/libraries/javascript/src/index.ts#L52-L55

Reply:
Thanks for the suggestion! Yes, I realised that this is an issue when I ran tests. Will push a fix

Reply:
Done!